### ISSUE TITLE
Correction factors for nitrogen transformation processes

### ISSUE THREAD

**Issue by kellsikellsi** [APSIMInitiative/ApsimX#7926]
Hello everyone,
Recently when I am trying to write codes about the nitrogen transformation processes， I found I cannot find the specific equations about all the correction factors, such as water and soil temperature factors in the nitrification and denitrification processes. On the information website, I only can know the correction factors' figures (e.g., https://www.apsim.info/documentation/model-documentation/soil-modules-documentation/soiln/). Could someone help to solve it? Thanks a lot!

**Comment by BrianCollinss** [comment]
The model documentation you shared the link of is related to APSIM Classic.

**Comment by kellsikellsi** [comment]
> 

yeah, I just want to compare these correction factors, but I don't know where I can find the equations...
